src/components/settings-menu/DevSection.tsx

```diff
@@ -206,24 +213,43 @@ const DevSection = () => {
     // from firebase first or we’re gonna keep getting them even after
     // clearing storage and before changing settings
     if (notificationSettings.length > 0) {
-      notificationSettings.forEach(wallet => {
-        removeNotificationSettingsForWallet(wallet.address);
-      });
+      return Promise.all(
+        notificationSettings.map(wallet =>
+          removeNotificationSettingsForWallet(wallet.address)
+        )
+      );
     }
+    return Promise.resolve();
   }, [notificationSettings]);
 
-  const clearLocalStorage = useCallback(async () => {
-    clearAllNotificationSettings();
+  const clearLocalStorage = async () => {
+    setLoadingStates(prev => ({ ...prev, clearLocalStorage: true }));
+
+    await clearAllNotificationSettings();
     await AsyncStorage.clear();
     clearAllStorages();
     addDefaultNotificationGroupSettings();
-  }, [clearAllNotificationSettings]);
+    setSettingsVersion(2);
```

Comment:
Are we setting this here to prevent the migration from running? Because it will have already run by the time we get to this code.

Reply:
Outdated, got rid of the version state